### ISSUE TITLE
Bug: Error on creating header when "list" clicked

### ISSUE THREAD

**Issue by gcingay** [Dolibarr/dolibarr#16083]
# Bug
Issue appeared after the migration to v13.0. 

Warning message appears when click to "List" under "Products" or "Services" or "3rd Party" in the menu. 

Warning: Cannot modify header information - headers already sent by (output started at /core/tpl/extrafields_list_array_fields.tpl.php:3) in main.inc.php on line 1202

Warning: Cannot modify header information - headers already sent by (output started at /core/tpl/extrafields_list_array_fields.tpl.php:3) in main.inc.php on line 1206

Warning: Cannot modify header information - headers already sent by (output started at /core/tpl/extrafields_list_array_fields.tpl.php:3) in main.inc.php on line 1207

## Environment
- **Version**: 13.0.0
- **OS**: Linux
- **Web server**: Apache
- **PHP**: cgi-fcgi 7.4.14
- **Database**: MySQL 
- **URL(s)**: /product/list.php?leftmenu=product&type=0

## Steps to reproduce the behavior
Warning message appears when click to "List" under "Products" or "Services" or "3rd Party" in the menu.

**Comment by eldy** [comment]
Do you have external modules enabled ? Try to disable all of them.

**Comment by gcingay** [comment]
Only Ecommerceng module was installed as external. It is disabled but still same error/warning message appears.

**Comment by JESSTOFUNK** [comment]
This is happening on everywhere a list is. I got the same error, no external modules installed.

I looked for configurations and found a section that is new in V13. It is called "ABOUT SECURITY" it said that my dolibarr was not in a production state with a warning mark. I have never seen that in other dollibar versions. Go to your conf.php and set "_$dolibarr_main_prod='**0**';_" to "_$dolibarr_main_prod='**1**';_". It has solved the error for me and warning mark is gone now, but I will look if there is another failures due to this.

**Comment by gcingay** [comment]
Much appreciate it. Thank you JESSTOFUNK. Now, it is working again.
  
  
   
    On 01/29/2021 7:49 PM JESSTOFUNK <notifications@github.com> wrote:
   
   
    
   
   
    
   
   
   This is happening on everywhere a list is. I got the same error, no external modules installed.
   I looked for configurations and found a section that is new in V13. It is called "ABOUT SECURITY" it said that my dolibarr was not in a production state with a warning mark. I have never seen that in other dollibar versions. Go to your conf.php and set "$dolibarr_main_prod='0';" to "$dolibarr_main_prod='1';". It has solved the error for me and warning mark is gone now, but I will look if there is another failures due to this.
